LICENSE

```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2014, NFLabs inc.
+   Copyright [yyyy] [name of copyright owner]
```

Comment:
This section is talking about "How to apply the Apache License to your work."
 
```
  APPENDIX: How to apply the Apache License to your work.        To apply the Apache License to your work, attach the following       boilerplate notice, with the fields enclosed by brackets "[]"       replaced with your own identifying information. (Don't include       the brackets!)  The text should be enclosed in the appropriate       comment syntax for the file format. We also recommend that a       file or class name and description of purpose be included on the       same "printed page" as the copyright notice for easier          identification within third-party archives. ```
 
So I think [yyyy] [name of copyright owner] is indeed correct.